src/System.Net.Http/src/System/Net/Http/Headers/HttpHeaderType.cs

```diff
@@ -0,0 +1,19 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+namespace System.Net.Http.Headers
+{
+    [Flags]
+    internal enum HttpHeaderType
```

Comment:
I'm pretty sure we had this conversation before, but presumably in theory we could mark all non public enums as :byte ? (With some care about what we were doing with them)